cli/cmd/trigger_sequence.go

```diff
@@ -0,0 +1,185 @@
+            - name: "release"
+*/
+
+var triggerSequenceCmd = &cobra.Command{
+	Use:     "sequence",
+	Aliases: []string{"sequence"},
+	Short:   "Triggers the execution of any sequence in a project",
+	Long: `Triggers the execution of any sequence in a project with an arbitrary name.
+The name of the sequence has to be provided as an argument to the command. The sequence name is used to identify the sequence to be triggered.
+`,
+	Example:      `keptn trigger sequence <sequence-name> --project=<project> --service=<service> --stage=<stage>`,
+	SilenceUsage: true,
+	Args: func(cmd *cobra.Command, args []string) error {
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return doTriggerSequence(sequence)
+	},
+}
+
+func doTriggerSequence(sequenceInputData sequenceStruct) error {
+	var endPoint url.URL
+	var apiToken string
+	var err error
+	if !mocking {
+		endPoint, apiToken, err = credentialmanager.NewCredentialManager(assumeYes).GetCreds(namespace)
+	} else {
+		endPointPtr, _ := url.Parse(os.Getenv("MOCK_SERVER"))
+		endPoint = *endPointPtr
+		apiToken = os.Getenv("MOCK_API_TOKEN")
+		// apiToken = ""
+	}
+	if err != nil {
+		return errors.New(authErrorMsg)
+	}
+
+	logging.PrintLog("Triggering sequence "+*sequenceInputData.Sequence+" in project "+*sequenceInputData.Project+" in stage "+*sequenceInputData.Stage+" service "+*sequenceInputData.Service, logging.InfoLevel)
+
+	api, err := internal.APIProvider(endPoint.String(), apiToken)
+	if err != nil {
+		return err
+	}
+
+	projectServices, err := api.ServicesV1().GetAllServices(*sequenceInputData.Project, *sequenceInputData.Stage)
+
+	if err != nil {
+		return fmt.Errorf("error while retrieving information for service %s: %s", *sequenceInputData.Service, err.Error())
+	}
+	if !ServiceInSlice(*sequenceInputData.Service, projectServices) {
+		return fmt.Errorf("could not start sequence because service %s has not been found in project %s", *sequenceInputData.Service, *sequenceInputData.Project)
+	}
+
+	deploymentEvent := keptnv2.DeploymentTriggeredEventData{
+		EventData: keptnv2.EventData{
+			Project: *sequenceInputData.Project,
+			Stage:   *sequenceInputData.Stage,
+			Service: *sequenceInputData.Service,
+		},
+	}
+
+	sdkEvent := cloudevents.NewEvent()
+	sdkEvent.SetID(uuid.New().String())
+	sdkEvent.SetType(keptnv2.GetTriggeredEventType(*sequenceInputData.Stage + "." + *sequenceInputData.Sequence))
+	sdkEvent.SetSource("https://github.com/keptn/keptn/cli#configuration-change")
+	sdkEvent.SetDataContentType(cloudevents.ApplicationJSON)
+	sdkEvent.SetData(cloudevents.ApplicationJSON, deploymentEvent)
+
+	eventByte, err := sdkEvent.MarshalJSON()
+	if err != nil {
+		return fmt.Errorf("Failed to marshal cloud event. %s", err.Error())
+	}
+
+	// fmt.Println(string(eventByte))
+
+	apiEvent := apimodels.KeptnContextExtendedCE{}
+	err = json.Unmarshal(eventByte, &apiEvent)
+	if err != nil {
+		return fmt.Errorf("Failed to map cloud event to API event model. %v", err)
```

Comment:
please start all error messages with a lower case, to make sure we are consistent with our error messages :)